src/App.tsx

```diff
@@ -1,24 +1,71 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  setInterval(() => {
+type Props = {
+
+};
+
+type State = {
+  isClockVisible: boolean,
+  clock: string,
+};
+
+class App extends React.Component<Props, State> {
```

Comment:
```suggestion
class App extends React.Component<{}, State> {
```
In this case you can remove your `type Props` above